comid-code-points.cddl

```diff
@@ -104,6 +98,12 @@ comid.serial-number = 3
 comid.ueid = 4
 comid.uuid = 5
 
+; reference-instance-claim-map
+comid.reference-claim-value = 6
+
+; endorsed-instance-claim-map
+comid.endorsed-claim-value = 7
```

Comment:
should be 6 as well since it's embedded in a different map

Reply:
Good point they are on a different map, so can be same! I will correct it!